CMakeLists.txt

```diff
@@ -215,3 +215,17 @@ if (test)
   add_dependencies(bdmclean testbdmclean)
 endif()
 
+#--------------------- build visualization projects ----------------------
+# build viz executable for small network test
+set(SMALL_NW_VIZ small-network-viz)
+add_executable(${SMALL_NW_VIZ} ${SOURCES} viz/small_network_visualization.cpp)
```

Comment:
`${SOURCES}` is not needed here - it builds all the object files that are already in `libbiodynamo.so`